### PR TITLE
Added basic classes

### PR DESCRIPTION
**Pull request by unres**

### CODE REVIEW

File: orangeChicken.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+public class orangeChicken extends Food{
```
Comment (panaman67) [inline orangeChicken.java]: Class name needs to be PascalCase. eg: OrangeChicken

File: orangeChicken.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  public orangeChicken{
```
Comment (panaman67) [inline orangeChicken.java]: Constructor change to match.

File: orangeChicken.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    id = 5;
+    name = "Orange Chicken";
+    price = 8.99f;
+  }
+
+  public String getDescription(){
+    return "fried chicken pieces coated in a sweet orange-flavored chili sauce";
+  }
+
```
Comment (panaman67) [inline orangeChicken.java]: Make sure spacing is 1 tab (4 spaces equivalent)

File: sweetSourChicken.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+public class sweetSourChicken extends food{
```
Comment (panaman67) [inline sweetSourChicken.java]: Same nits as OrangeChicken

**Review by panaman67 — CHANGES_REQUESTED** [review]
All style stuff, but important

**Comment by unres** [comment]
Fixed!

**Review by panaman67 — APPROVED** [review]

**Comment by panaman67** [comment]
LGTM @unres ! Merging now

**Pull request merged**